android/demoapps/javademo/src/main/java/com/azuresamples/quickauthandroidapp/util/ByteCodeUtil.java

```diff
@@ -34,10 +34,10 @@ public class ByteCodeUtil {
    * @param base64 base64 string
    * @return Bitmap
    */
-  public static Bitmap base642Bitmap(String base64) {
+  public static Bitmap base64ToBitmap(String base64) {
     if (TextUtils.isEmpty(base64)) return null;
     try {
-      byte[] decode = Base64.decode(base64, Base64.DEFAULT);
+      byte[] decode = Base64.decode(base64, Base64.NO_WRAP);
       return BitmapFactory.decodeByteArray(decode, 0, decode.length);
     } catch (Exception e) {
```

Comment:
You may want to catch `Throwable` instead of `Exception` because `Throwable` contains `OutOfMemoryError`.

Reply:
Since it is just a demo app, it is not that necessary to change it.

Reply:
Thanks, updated